frontend/apps/app/app/(app)/app/organizations/[organizationId]/settings/general/DeleteConfirmationModal.tsx

```diff
@@ -0,0 +1,86 @@
+'use client'
+
```

Comment:
Is it necessary?

I would prefer that you do not set `'use client'` unless you have a reason to do so, as there are many advantages to having it as a Server Component!

ref: https://nextjs.org/docs/app/building-your-application/rendering/server-components

Reply:
It was not necessary.🙏
I deleted this.
https://github.com/liam-hq/liam/pull/1420/commits/a3129177cbd346fb284e1a57a1575789199c036b